### DISCUSSION TITLE
Turn on status.

### DISCUSSION THREAD

**Discussion by hochzmoravy** [alandtse/tesla#138] [Q&A]
Is there a trigger entity that can be used for automation?
I want my garage to open when I get in my car. I've tried different entities but none of them work the way I want them to.

**Comment by JevVan** [comment]
Yes, I just was doing the same thing and noticed the Parking brake doesn't register as off until I'm well out of the driveway.  I'm not sure if this is intentional by tesla, or if it's a slow refresh / update interval?
